### PR TITLE
Add areas to location

### PR DESCRIPTION
**Pull request by dbrgn**
This key can be used to specify a list of areas that make up your physical location. Each area has a required size (in square meters), and an optional name and description.

Replaces #71. (See discussion in that MR for context.)

### CODE REVIEW

**Comment by dbrgn** [comment]
@SpaceApi/core if I understood you correctly during today's meeting, this was the proposal that we converged on. Do you agree with that?

(The old PR was still against schema v14-draft, btw 😂)

**Review by s3lph — APPROVED** [review]
LGTM 🚀

File: CHANGELOG.md
Comment (rnestler) [inline CHANGELOG.md:25]: The change in order seems unnecessary, but I don't care that much.

**Review by rnestler — APPROVED** [review]
LGTM as well.

**Comment by rnestler** [comment]
@dbrgn Whats holding the merge back?

File: CHANGELOG.md
Comment (dbrgn) [reply CHANGELOG.md:25]: I think that's an auto-merge thing by git, but I'm not sure.

**Comment by dbrgn** [comment]
> @dbrgn Whats holding the merge back?

4 hours ago, it was the missing second review 😂 Once you added your review, you could have hit the "merge" button.

**Comment by rnestler** [comment]
> 4 hours ago, it was the missing second review 😂 Once you added your review, you could have hit the "merge" button.

Ah I think we agreed that 1 review is enough if the change comes from a core member anyways.

**Pull request merged**
